src/lexer.rs

```diff
@@ -114,6 +116,7 @@ impl<'a> ParseSession<'a> {
     }
 
     pub fn advance(&mut self) {
+        log::trace!("{}", self.slice());
```

Comment:
do you want to keep this here? if this has no performance implications its ok - I'm not very familiar with our logging approach